client/src/pages/Messages/ActiveChat/ActiveChatHeader/useStyles.ts

```diff
@@ -0,0 +1,46 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  activeChatHeader: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    height: 125,
+    paddingRight: theme.spacing(3),
+    borderBottom: '1px solid #eee',
+    [theme.breakpoints.down('sm')]: {
+      height: '82px',
+    },
+    [theme.breakpoints.down('xs')]: {
+      width: '100vw',
+    },
+  },
+  mobileActiveChatHeader: {
+    backgroundColor: '#fff',
+
+    [theme.breakpoints.up('sm')]: {
+      display: 'none',
+    },
+  },
+  content: {
+    display: 'flex',
+    alignItems: 'center',
+    marginLeft: theme.spacing(3),
+  },
+  username: {
+    fontSize: 20,
```

Comment:
You can assign the 'variant' prop to `Typography` component to change font sizes and weight where necessary :)

Eg: `<Typography variant="h5">`

Ditto throughout!

Reply:
Applied